src/main/java/core/basesyntax/Application.java

```diff
@@ -2,7 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        Lottery lottery = new Lottery();
-        System.out.println(lottery.getRandomBall());
+        Lottery lottery1 = new Lottery();
+        Lottery lottery2 = new Lottery();
+        Lottery lottery3 = new Lottery();
+        Lottery [] lotteries = {lottery1,lottery2,lottery3};
+        for (Lottery lottery:lotteries) {
+            System.out.println(lottery.getRandomBall());
+        }
+
```

Comment:
there is no need to create 3 lotteries

lottery is just a service that can be reused
we can just call method in a loop 3 times